frontend/src/templates/Field/FieldContainer.tsx

```diff
@@ -59,7 +59,6 @@ export const FieldContainer = ({
   return (
     <FormControl
       isRequired={schema.required}
-      isDisabled={schema.disabled}
```

Comment:
any reason why we removed this prop from the base container? this container should be the ultimate controller of field state instead of being in the children imho

Reply:
I initially used the base container's isDisabled, but I realised this also disables the form label e.g.
<img width="765" alt="image" src="https://user-images.githubusercontent.com/56983748/184569702-1165ba68-ac6f-4323-9d09-011d102915df.png">
which is different from the form design system's [disabled field](https://www.figma.com/file/u00dlQLsaQfvqv6ot5ZrJZ/Form-Design-System?node-id=2153%3A4682)

Reply:
we could update the design for a disabled form label instead i think. This is changing logic to fulfil styling, which may introduce subtle bugs in the future

Reply:
ah i see ok makes sense! will look into it